tools/vardictjava.cwl

```diff
@@ -5,24 +5,28 @@ requirements:
   - class: ShellCommandRequirement
   - class: InlineJavascriptRequirement
   - class: ResourceRequirement
-    ramMin: 18
-    coresMin: 4
+    ramMin: ${return inputs.ram * 1000}
+    coresMin: $(inputs.cpus)
   - class: DockerRequirement
-    dockerPull: 'kfdrc/vardict:1.5.8'
+    dockerPull: 'migbro/vardict:1.7.0'
```

Comment:
can we move this image to `kfdrc` docker hub?

Reply:
ah! good catch!